cmd/format.go

```diff
@@ -1,22 +1,43 @@
 package cmd
 
 import (
-	"fmt"
-
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+
+	"github.com/elastic/elastic-package/internal/formatter"
+	"github.com/elastic/elastic-package/internal/packages"
 )
 
+const failFastFlagName = "fail-fast"
```

Comment:
Not for this PR but in https://github.com/elastic/elastic-package/pull/44 I introduced an `internal/cmd` package for various common utilities/consts/etc. we need across multiple command implementations. We could consider moving this const there later.

Reply:
Ref: https://github.com/elastic/elastic-package/pull/44/files#diff-e343a6b946d6d699b7b341b437c58e4b

Reply:
I looked into IDE and saw the problem you spotted. If we follow your idea with moving the Compose.. to `internal`, the `cmd` package would be cleaner.

As this is strongly related with https://github.com/spf13/cobra , I suggest to name the package `cobraext`. WDYT?